.idea/runConfigurations.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What is this file for, and why do we want Android?

Reply:
I don't know the answer to either question; I think it got made when I had to do a hard reset and update to 2021.2.